package.json

```diff
@@ -10,7 +10,7 @@
     "lint": "yarn run lint:base -- --fix",
     "lint:base": "eslint . --ext .js, .jsx",
     "modules:clean": "rm -rf node_modules",
-    "prepublish": "yarn run build",
+    "install": "yarn run build",
```

Comment:
for npm v5 compatibility. prepublish will no longer run automatically once v5 rolls out.